easybuild/easyconfigs/a/ASE/ASE-3.17.0-intel-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,60 @@
+easyblock = 'PythonBundle'
+
+name = 'ASE'
+version = '3.17.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('Tkinter', '%(pyver)s'),
+    ('matplotlib', '3.0.3', '-Python-%(pyver)s'),
+]
+
+use_pip = True
+
```

Comment:
Here your need:
```
# required because we're building Python packages using Intel compilers on top of Python built with GCC
check_ldshared = True
```

Reply:
OK, I will update that.  What does it really do?

Reply:
It makes a small rewrite of the share-lib link command to make sure it works for the active compiler.

Reply:
OK, then I do not think it is needed.  ASE is pure Python, it does not compile anything but is at toolchain level because it depends on SciPy and matplotlib.

Reply:
Not quite true. There is ASE/3.17.0-Python-3.7.2/lib/python3.7/site-packages/markupsafe/_speedups.cpython-37m-x86_64-linux-gnu.so
which needs the check_ldshared flag, but technically speaking it's only needed for the markupsafe extension.

Reply:
Ah!  I hadn't thought about the other packages in the bundle.  Of course.  I'll leave it in.

Reply:
The `check_ldshared` can actually be set for `markupsafe` specifically, but it doesn't matter too much (maybe other extensions get added later that also require it)